src/dvclive/live.py

```diff
@@ -151,7 +152,12 @@ def _init_dvc(self):
             self._include_untracked.append(self.dir)
 
     def _init_studio(self):
-        if not any((os.getenv(STUDIO_TOKEN, None), os.getenv(DVC_STUDIO_TOKEN, None))):
+        self._studio_token = os.getenv(STUDIO_TOKEN, None) or os.getenv(
+            DVC_STUDIO_TOKEN, None
+        )
+        if self._dvc_repo and not self._studio_token:
+            self._studio_token = self._dvc_repo.config.get("studio", {}).get("token")
```

Comment:
can DVCLive now work without DVC repo? Outside a repo? 

We still could read the global / user config in that case I think.

Reply:
DVCLive will try to create a DVC repo now (see #539). It should only fail if it's outside a Git repo or the Git repo is empty. That's a valid use case but handling it is outside this PR IMO (to start, Studio would need to handle a non-Git project).

Reply:
On 2nd thought, it shouldn't be that hard to support in the use case where I have a repo and project but want to run a specific experiment outside of the repo in some cloud managed cloud environment.

Reply:
Done

Reply:
@daavoo What do you think about this scenario? I think it's disabled now if there's no DVC repo, but should we allow it for this kind of use case?

Reply:
> @daavoo What do you think about this scenario? I think it's disabled now if there's no DVC repo, but should we allow it for this kind of use case?

Sorry, I am not fully understanding the scenario. Could you elaborate?

Reply:
Let's say I have an existing Git repo and Studio project, but I run a specific experiment disconnected from my repo (in colab, databricks, etc.). I could set the token and repo url and still send live metrics, except that I think we currently disable this if no DVC repo is found. Should we allow it?